### PR TITLE
profile support: validate_content parameter is not supported in 2015-04-05 data storage sdk

### PR DESCRIPTION
**Pull request by bganapa**
Conditionally removing the use of validate_content for the older api

---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] The PR has modified HISTORY.rst with an appropriate description of the change (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

### Command Guidelines

- [ ] Each command and parameter has a meaningful description.
- [ ] Each new command has a test.

(see [Authoring Command Modules](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules))

### CODE REVIEW

**Comment by bganapa** [comment]
cc: @derekbekoe

**Comment by derekbekoe** [comment]
@bganapa These changes look good to me but what about the File commands in `file.py`?
There's validate_content in there as well.

**Comment by bganapa** [comment]
@derekbekoe File is not supported in azurestack , So I left it as such..

**Comment by bganapa** [comment]
@derekbekoe addressed your comments on file.py, added conditions there as well

File: src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/_params.py [hunk 1]
```diff
@@ -421,9 +427,11 @@ def register_blob_source_uri_arguments(scope):
     c.reg_arg('destination', options_list=('--destination', '-d'))
 
     with c.arg_group('Download Control') as group:
-        group.reg_arg('validate_content')
         group.reg_arg('max_connections')
 
+        with VersionConstraint(ResourceType.DATA_STORAGE, min_api='2016-05-31') as vc:
```
Comment (bganapa) [inline src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/_params.py]: @derekbekoe  Can you please verify this line 432? Does this need to start without any white spaces in the beginning?

**Pull request merged**